client/app/assets/styles/variables.js

```diff
@@ -205,11 +208,30 @@ module.exports = {
   '--MenuSection_fontSizeTitle': fontSizeSmaller,
   '--MenuSection_paddingOffScreenVertical': pxToEms(10, 12), // eslint-disable-line no-magic-numbers
   '--MenuSection_paddingOffScreenHorizontal': pxToEms(24, 12), // eslint-disable-line no-magic-numbers
+  '--MenuSection_paddingOffScreenHorizontalTablet': pxToEms(36, 12), // eslint-disable-line no-magic-numbers
+  '--MenuSection_iconMargin': pxToEms(9, 12), // eslint-disable-line no-magic-numbers
 
   '--MobileMenu_labelPaddingVertical': '18px',
   '--MobileMenu_labelPaddingHorizontal': '18px',
   '--MobileMenu_offscreenMenuWidth': '288px',
   '--MobileMenu_offscreenHeaderItemHeight': '44px',
+  '--MobileMenu_offscreenFooterBackgroundColor': '#F7F7F7',
```

Comment:
This could be added to variables as backgroundColorGrey (or something like that) to keep track all the shades of grey